.github/workflows/main.yml

```diff
@@ -117,29 +117,30 @@ jobs:
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v1
 
-      - name: Cache layers
-        id: cache-buildx
-        uses: actions/cache@v2
-        with:
-          path: /tmp/.buildx-cache
-          key: ${{ runner.os }}-buildx-backend-${{ hashFiles('authorizer-app-backend/Dockerfile', '**/*.gradle.kts', 'gradle.properties', 'authorizer-app-backend/src/main/**') }}
-          restore-keys: |
-            ${{ runner.os }}-buildx-backend-
-
-      - name: Cache parameters
-        id: cache-parameters
-        run: |
-          if [ "${{ steps.cache-buildx.outputs.cache-hit }}" = "true" ]; then
-            echo "::set-output name=cache-to::"
-          else
-            echo "::set-output name=cache-to::type=local,dest=/tmp/.buildx-cache-new,mode=max"
-          fi
+      # - name: Cache layers
+      #   id: cache-buildx
+      #   uses: actions/cache@v2
+      #   with:
+      #     path: /tmp/.buildx-cache
+      #     key: ${{ runner.os }}-buildx-backend-${{ hashFiles('authorizer-app-backend/Dockerfile', '**/*.gradle.kts', 'gradle.properties', 'authorizer-app-backend/src/main/**') }}
+      #     restore-keys: |
+      #       ${{ runner.os }}-buildx-backend-
+
+      # - name: Cache parameters
+      #   id: cache-parameters
+      #   run: |
+      #     if [ "${{ steps.cache-buildx.outputs.cache-hit }}" = "true" ]; then
+      #       echo "::set-output name=cache-to::"
+      #     else
+      #       echo "::set-output name=cache-to::type=local,dest=/tmp/.buildx-cache-new,mode=max"
+      #     fi
 
       - name: Build backend docker
         uses: docker/build-push-action@v2
         with:
           context: ./
           file: ./authorizer-app-backend/Dockerfile
+          platforms: linux/amd64,linux/arm64
```

Comment:
This only works if `load: false`. If you want to specifically test the `arm64` build, you could have two separate build steps:
```yaml
       - name: Cache parameters
         id: cache-parameters
         run: |
           if [ "${{ steps.cache-buildx.outputs.cache-hit }}" = "true" ]; then
             echo "::set-output name=cache-to::"
           else
             echo "::set-output name=cache-to::type=local,dest=/tmp/.buildx-cache-new,mode=max"
           fi
           if [ "${{ github.event_name == 'pull_request' }}" = "true" ]; then
             echo "::set-output name=push::false"
           else
             echo "::set-output name=push::true"
           fi
 
      - name: Build backend docker
        uses: docker/build-push-action@v2
        with:
          context: ./
          file: ./authorizer-app-backend/Dockerfile
          platforms: linux/amd64,linux/arm64
          cache-from: type=local,src=/tmp/.buildx-cache
          cache-to: ${{ steps.cache-parameters.outputs.cache-to }}
          push: ${{ steps.cache-parameters.outputs.push }}
        tags: ${{ steps.docker_meta.outputs.tags }}
        # Use runtime labels from docker_meta_backend as well as fixed labels
        labels: |
          ${{ steps.docker_meta.outputs.labels }}
          maintainer=Joris Borgdorff <joris@thehyve.nl>, Nivethika Mahasivam <nivethika@thehyve.nl>, Pauline Conde <pauline.conde@kcl.ac.uk>
          org.opencontainers.image.description=RADAR-base rest sources authorizer backend application
          org.opencontainers.image.authors=Joris Borgdorff <joris@thehyve.nl>, Nivethika Mahasivam <nivethika@thehyve.nl>, Pauline Conde <pauline.conde@kcl.ac.uk>
          org.opencontainers.image.vendor=RADAR-base
          org.opencontainers.image.licenses=Apache-2.0
 
      # will use the internal cache from the previous build step, and load it into the current memory
      - name: Build backend docker locally
        uses: docker/build-push-action@v2
        with:
          context: ./
          file: ./authorizer-app-backend/Dockerfile
          load: true
        tags: ${{ steps.docker_meta.outputs.tags }}
```

The separate push step can be removed then.

Reply:
Okay thanks!